app/views/licences/tabs/returns.njk

```diff
@@ -19,36 +19,56 @@
   {% endif %}
 {% endmacro %}
 
-<table class="govuk-table">
-  <h2 class="govuk-heading-l">Returns</h2>
-  {% if returns.length > 0 %}
-    <thead class="govuk-table__head">
-    <tr class="govuk-table__row">
-      <th scope="col" class="govuk-table__header">Return reference and dates</th>
-      <th scope="col" class="govuk-table__header">Purpose and description</th>
-      <th scope="col" class="govuk-table__header">Due date</th>
-      <th scope="col" class="govuk-table__header">Status</th>
-    </tr>
-    </thead>
-    <tbody class="govuk-table__body">
-    {% for return in returns %}
-      <tr class="govuk-table__row govuk-body-s">
-        <th class="govuk-table__cell">
-          <a href="/return/internal?returnId={{ return.id }}" class="govuk-link">{{ return.reference }} </a>
-          <p class="govuk-body-s"> {{ return.dates }}</p>
-        </th>
-        <td class="govuk-table__cell">{{ return.purpose }} <p class="govuk-body-s"> {{ return.description }}</p></td>
-        <td class="govuk-table__cell">{{ return.dueDate }}</td>
-        <td class="govuk-table__cell">{{ formatStatus(return.status) }}</td>
-      </tr>
-    {% endfor %}
-    </tbody>
-  {% else %}
-    <p>No returns found</p>
-  {% endif %}
-</table>
+<h2 class="govuk-heading-l">Returns</h2>
+
+{% if noReturnsMessage %}
+  <p> {{ noReturnsMessage }} </p>
+{% else %}
+
+  {% macro referneceColum(return) %}
+    <a href="/return/internal?returnId={{ return.id }}" class="govuk-link">{{ return.reference }} </a>
+    <p class="govuk-body-s"> {{ return.dates }}</p>
+  {% endmacro %}
+
+  {% set returnRows = [] %}
+
+  {% for returnItem in  returns %}
+    {% set returnRows = (returnRows.push([
+      {
+        html: referneceColum(returnItem)
+      },
+      {
+        html:  returnItem.purpose +  "<p class=\"govuk-body-s\">" + returnItem.description + "</p>"
+      },
+      {
+        text: returnItem.dueDate
+      },
+      {
+        html: formatStatus(returnItem.status)
+      }
+    ]), returnRows) %}
+  {% endfor %}
+
+  {{ govukTable({
+    head: [
+      {
+        text: 'Return reference and dates'
+      },
+      {
+        text: 'Purpose and description'
+      },
+      {
+        text: 'Due date'
+      },
+      {
+        text: 'Status'
+      }
+    ],
+    rows: returnRows
+  }) }}
+{% endif %}
 
-{% if returns and pagination.numberOfPages > 1 %}
+{% if hasReturns and pagination.numberOfPages > 1 %}
```

Comment:
Thinking about it, if the licence doesn't have any returns then `pagination.numberOfPages` will equal 0. So, we don't actually need the double check.

```suggestion
{% if pagination.numberOfPages > 1 %}
```

Reply:
Good spot i missed then when moving the message into the presenter.